packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -182,4 +204,36 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
         null
       );
   }
+
+  protected _onRenderMenuIcon = (props: IButtonProps): JSX.Element => {
```

Comment:
Interesting, in VSTS (at least for parts of it) we are moving in the opposite direction, away from the lambda-class-fields towards `@autobind` since it expresses your intention a bit more explicitly. 

@dzearing thoughts?